modules/phase_field/src/action/GrandPotentialKernelAction.C

```diff
@@ -65,7 +65,9 @@ validParams<GrandPotentialKernelAction>()
   parameters.addParam<bool>("implicit", true, "Whether kernels are implicit or not");
   parameters.addParam<bool>(
       "use_displaced_mesh", false, "Whether to use displaced mesh in the kernels");
-  parameters.addParam<bool>("anisotropic", false, "Set to true if the diffusivity is a tensor");
+  MultiMooseEnum anisotropy("true false", "false");
```

Comment:
if you order this `false true` or explicitly set `true=1 false=0` you can use an implicit bool cast to check the value...

Reply:
How would that look? On line 269 I have `if (aniso[i] == "true")`. If I reference `aniso[i]` it changes types to `std::string`, but `aniso.contains` function would search all of the entries, not just entry `i`.

Reply:
Okay, I just found the `get` function.